src/triage/component/postmodeling/contrast/README.md

```diff
@@ -1,38 +1,196 @@
+       top-n features with their standard deviation. This is highly informative
+       in RandomForest models where each three can have different relevant
+       features. The function needs the user to pass the project path to the `path`
+       argument. More details in the [Configuration File](#configuration-file) section.
+
+     - `plot_feature_group_average_importance` makes the same excercise, but it
+       aggreagates (averages) feature importance metrics to the feature group
+       level and plots the relevance of each feature group. 
+
+ - **Model Matrix characteristics**: _These function need the user to pass
+   Triage's project to the `path` argument. More details in the [Configuration
+   File](#configuration-file) section._
+   
+     - `cluster_correlation_features` shows a correlation matrix ordered by the
+         correlation between features. This plot can be subsetted by a feature
+         group and explore the correlation in that set of the feature space. 
+  
+     - `cluster_correlation_sparsity` renders an image with the prediction matrix
+       colored by their data availabilty. This plot can shows different
+       consequences of data imputation and help the user to visualize the
+       zero-only features (it happens with individual constant features). 
+  
+     - `plot_feature_distribution` plots the distribution of the top_n features
+       comparing the positive and negative labeled entities. A three-column plot
+       is rendered, the first two corresponding to the individual label plots,
+       and the third one corresponding to both. 
+
+ - **Model Metrics**:
+   - `plot_ROC` plots the AUC-ROC curve plot for the selected model. This is
+     just a wrapper of the `sklearn` library function to be compatible with
+     triage.
+
+   - `plot_precision_recall_n` renders the precision/recall curves for the
+     desired model
+     
+   - `plot_recall_fpr_n` plots the false positive rate against the threshold of
+     absolute population (another way of exploring recall).
+
+ - **Error Analysis**: Error analysis is a powerful tool to identify where the
+   model is making classification mistakes. Error analysis involves several
+   other hidden functions, but it can be completely run using the
+   `error_analysis` function. This function will label the errors and use
+   a `DecisionTreeClassifier` to classify the errors and print the modeled
+   tree. This function needs a parameters object defined in the configuration
+   file. More details in the [Configuration File](#configuration-file)
+   section.
+
+ - **Crosstabs Analysis**: `crosstabs_ratio_plot` will plots the mean ratio for
+   all relevant features in the model. This allows a good comparison of
+   true/false predicted groups and get the key differences between groups in
+   the feature space. The function expects that the user run
+   [crosstabs](https://github.com/dssg/postmodel-analysis/tree/sqlqueries/postmodel/crosstabs)
```

Comment:
We're linking to postmodeling crosstabs in the other repo? What about the crosstabs in this repo? Furthermore, the linked repo is private. If what's in there is necessary for this to run, it should be in this public repo.

Reply:
I talked with @saleiro about it. There's an open issue about it [#586]. We should discuss how to integrate both libraries successfully.

Reply:
I agree with @thcrock , crosstabs should move to this repo

Reply:
Fixed!